modules/openxr/action_map/openxr_binding_modifier.h

```diff
@@ -0,0 +1,70 @@
+/**************************************************************************/
+/*  openxr_binding_modifier.h                                             */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
+/* Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.                  */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#ifndef OPENXR_BINDING_MODIFIER_H
+#define OPENXR_BINDING_MODIFIER_H
+
+#include "../action_map/openxr_action.h"
+#include "core/io/resource.h"
+
+// Part of implementation for:
+// https://registry.khronos.org/OpenXR/specs/1.1/html/xrspec.html#XR_KHR_binding_modification
+
+class OpenXRInteractionProfile;
+class OpenXRIPBinding;
+
+class OpenXRBindingModifier : public Resource {
+	GDCLASS(OpenXRBindingModifier, Resource);
+
+private:
+protected:
+	friend class OpenXRInteractionProfile;
+	friend class OpenXRIPBinding;
+
+	static void _bind_methods();
+
+	GDVIRTUAL0RC(bool, _record_on_binding)
+	GDVIRTUAL0RC(String, _get_description)
+	GDVIRTUAL0R(PackedByteArray, _get_ip_modification)
```

Comment:
These feel like they should be required? So:

```suggestion
	GDVIRTUAL0RC_REQUIRED(bool, _record_on_binding)
	GDVIRTUAL0RC_REQUIRED(String, _get_description)
	GDVIRTUAL0R_REQUIRED(PackedByteArray, _get_ip_modification)
```
This will make it an error if `GDVIRTUAL_CALL()` can't find the method, and mark them as required in the `extension_api.json`.

This won't have any effect on the sub-classes within Godot that override these methods; only for GDExtension classes that do.